ui/ducks/send/send.js

```diff
@@ -1298,6 +1302,7 @@ export function resetSendState() {
       await disconnectGasFeeEstimatePoller(
         state[name].gas.gasEstimatePollToken,
       );
+      removePollingTokenFromAppState(state[name].gas.gasEstimatePollToken);
```

Comment:
Let's make sure that `resetSendState` is dispatched on beforeunload in the send component